### ISSUE TITLE
VanillaCubic and CustomCubic don't render blocks from Y=0-15

### ISSUE THREAD

**Issue by BaccarWozat** [Barteks2x/CubicChunks#248]
Builds 631 & 633, Forge 2315.

Both VanillaCubic and CustomCubic seem to now have problems with chunks from Y=0-15. Most of the chunks are invisible and impassable. A small number are invisible and may be flown through. A very small number are visible and solid.

This has been tested in several worlds now with different custom settings or even default. It has also been tested even with utility mods like Waila, JEI, More Overlays, etc. removed, just in case.

I reloaded the map with build 627 and the chunks all reappeared.

No crash but here's the log from the first one anyway: https://pastebin.com/ZqhSB1cr

![2017-07-12_15 05 49](https://user-images.githubusercontent.com/8357501/28137700-c260d0c6-6713-11e7-9a4f-7fd20ea80b62.png)
![2017-07-12_22 54 09](https://user-images.githubusercontent.com/8357501/28150091-fd457ec8-6757-11e7-8052-5929a72c367d.png)

**Comment by Niko-sk2x** [comment]
So it looks like for some reason they can't be sent to client.

**Comment by BaccarWozat** [comment]
With build 633 you can no longer see through them, but the blocks are unbreakable and still can't be entered.

**Comment by Niko-sk2x** [comment]
633 didn't change anything there.

**Comment by BaccarWozat** [comment]
I noticed this behavior has now surfaced in CustomCubic as well (hopefully it isn't all maps, but the one I was testing has this). I was able to go into the area, but some chunks wouldn't let me in and others would.

I also teleported to Y<0 and everything was as normal... for CustomCubic (haven't tested VanillaCubic for that).

Obviously more testing of both map types will be needed.

![2017-07-12_22 54 09](https://user-images.githubusercontent.com/8357501/28150091-fd457ec8-6757-11e7-8052-5929a72c367d.png)
